getting_started/setup/setup_up_2_board.rst

```diff
@@ -16,20 +16,15 @@ version is not currently supported.
 Setup Instructions
 ******************
 
-1. Flash BIOS version 3.6 onto the Up squared board.
+1. Flash BIOS version 4.0 onto the Up squared board.
 ======================================================
 
-BIOS v3.6 added the audio OEM key for SOF. The BIOS main menu
-will show UP-APL01 R3.6.
+The BIOS main menu will show UP-APL01 R4.0.
 
-* Download the `BIOS <https://git-amr-4.devtools.intel.com/gerrit/gitweb?p=otc_audio-board-integration.git;tflink=projects.otc_audio/scm.Board_Integration>`__.
-
-  .. todo::
-   
-     this link needs to be updated to something accessible externally
+* Download the `BIOS <https://downloads.up-community.org/download/up-squared-uefi-bios-v4-0/>`__.
```

Comment:
@intelkevinputnam Can you help me understand the double '_ _' at the end of the link? It doesn't seem to fit with the [guidelines](https://thesofproject.github.io/latest/contribute/doc_guidelines.html#internal-cross-reference-linking). I left it alone b/c I wasn't sure what it was there for!

Reply:
@stripes416 the link works without the__ 
https://downloads.up-community.org/download/up-squared-uefi-bios-v4-0/